roofit/roofitcore/inc/RooFuncWrapper.h

```diff
@@ -53,6 +80,18 @@ class RooFuncWrapper final : public RooAbsReal {
          }
          _params.add(*param);
       }
+
+      // Build a wrapper over the derivative to hide clad specific types such as 'array_ref'.
+      // disable clang-format for making the following code unreadable.
+      // clang-format off
+      std::string nParams = std::to_string(_params.size());
+      std::string dWrapper = "void " + wrapperName + "(double* in, double* out) {\n"
+                             "  clad::array_ref<double> cladOut(out, " + nParams + ");\n"
+                             "  " + gradName + "(in, cladOut);\n"
+                             "}";
```

Comment:
```suggestion
      std::stringstream dWrapperStream;
      dWrapperStream << "void " << wrapperName << "(double* in, double* out) {\n"
                             "  clad::array_ref<double> cladOut(out, " << _params.size() << ");\n"
                             "  " << gradName << "(in, cladOut);\n"
                             "}";
```
It's more efficient to use stringstream in these situations. Then you also don't need to turn the numbers into strings before.